<launch>
  <node name="target_follower" pkg="robosub" type="basic_target_follower_node">
    <param name="target_topic" value="/target" />
    <param name="kSpeed" value="0.1" type="double" />
    <param name="kAlt" value="0.1" type="double" />
    <param name="kYaw" value="0.1" type="double" />
  </node>
</launch>
